docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -53,16 +55,8 @@ associated command-line flags.
 ## Relationship to Dynamic Kubelet Config
 
 If you are using the [Dynamic Kubelet Configuration](/docs/tasks/administer-cluster/reconfigure-kubelet)
-feature, the configuration provided via `--init-config-dir` will be considered
-the "last known good" configuration by the automatic rollback mechanism.
-
-Note that the layout of the files in the `--init-config-dir` mirrors the layout
-of data in the ConfigMaps used for Dynamic Kubelet Config; the file names are
-the same as the keys of the ConfigMap, and the file contents are JSON or YAML
-representations of the same structures. Today, the only pair is 
-`kubelet:KubeletConfiguration`, though more may emerge in the future.
-See [Reconfigure a Node's Kubelet in a Live Cluster](/docs/tasks/administer-cluster/reconfigure-kubelet)
-for more information.
+feature, the combination of flags and configuration provided via `--config` 
```

Comment:
The `combination` means `Note that command line flags which target the same value as a config file will override that value.` mentioned above, right?

Reply:
Yes